src/test/ui/or-patterns/exhaustiveness-unreachable-pattern.rs

```diff
@@ -49,9 +49,9 @@ fn main() {
         _ => {}
     }
     match [0; 2] {
-        [0
+        [0 //.
```

Comment:
What's with the `//.`s?

Reply:
Ah I can remove them, it's because my editor runs rustfmt on save and that merged the lines. Except I broke the lines that way to ensure the error was located correctly

Reply:
Fixed